test/Table_Tests/src/Excel_Spec.enso

```diff
@@ -518,24 +518,24 @@ spec =
             table = xlsx_sheet.read (File_Format.Excel (Sheet 1))
             check_table table
 
-            table_2 = xlsx_sheet.read (File_Format.Excel (Sheet 1 (table.length - col_a.length)))
-            table_2.length . should_equal col_a.length
+            table_2 = xlsx_sheet.read (File_Format.Excel (Sheet 1 (table.row_count - col_a.length)))
+            table_2.row_count . should_equal col_a.length
             check_table <| table_2
 
         Test.specify "should let you read by sheet name" <|
             table = xlsx_sheet.read (File_Format.Excel (Sheet "Sheet1"))
             check_table table
 
-            table_2 = xlsx_sheet.read (File_Format.Excel (Sheet "Sheet1" (table.length - col_a.length)))
-            table_2.length . should_equal col_a.length
+            table_2 = xlsx_sheet.read (File_Format.Excel (Sheet "Sheet1" (table.row_count - col_a.length)))
+            table_2.row_count . should_equal col_a.length
```

Comment:
This shows perfectly why I'm not sure if the `length` method is a good idea - indeed conceptually it seems to fit more `row_count` than `column_count`, even if `at` is used for indexing columns, not rows. It was already implemented the other way for in memory table.

We can keep it, but I think we should rethink what are the benefits of that - as it can easily cause confusion.

Reply:
Agree - I think removing length is the cleanest approach.

Reply:
Renamed to `column_count` so now we have both `row_count` and `column_count`.